frontend/www/schools/activity.php

```diff
@@ -0,0 +1,4 @@
+<?php
+require_once(dirname(__DIR__, 2) . '/server/bootstrap_smarty.php');
+$constant = 'constant';
+$smarty->display("{$constant('OMEGAUP_ROOT')}/templates/course.activity.tpl");
```

Comment:
qué tenía de malo

```suggestion
$smarty->display(OMEGAUP_ROOT . '/templates/course.activity.tpl');
```

?

Reply:
De la forma anterior me aparece un error en `psalm`:
```
ERROR: MixedOperand - frontend/www/course/scoreboard.php:4:18 - Left operand cannot be mixed
$smarty->display(OMEGAUP_ROOT . '/templates/arena.scoreboard.tpl');
```

Con la interpolación ya evito esos errores. Lo regresaré como estaba originalmente, pero tengo que averiguar como arreglar el error.